modules/plot/main/coffee/Utils.coffee

```diff
@@ -204,7 +204,6 @@ search = (array, find, lookup) ->
 findLabel = (array, find, interpolate, interpolateValues) ->
   i = search(array, find, (d) -> d.x)
   if i > -1
-    atEdge = i is 0 or i is array.length - 1
```

Comment:
I'm not sure why we did this, I can't think of a reason for it other than something unrelated to `LineSeries`